lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
Yeah, my main worry is having a bunchof back-and-forth API calls to grab all data and it totally killing the processing time due to all the HTTP requests. I'm also not 100% sure I know _what_ the user avatar is and how it differs from the profile picture (which I think was just used for MCC stuff)...